src/renderer/createStore.js

```diff
@@ -21,8 +21,11 @@ export default ({ state, dbMiddleware }: Props) => {
     middlewares.push(dbMiddleware);
   }
 
-  const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
-  const enhancers = composeEnhancers(applyMiddleware(...middlewares));
+  // const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
```

Comment:
Why? and if not needed, why not drop it

Reply:
That's an artefact. In fact, I tried to improve the redux configuration to fix the issue with the extension but I didn't test it enough to be sure that the changes fix something. I didn't want to push this change, it should stay stashed from my laptop. I'm going to remove this change.

Reply:
OK

Reply:
Removed from [0266e4e93ff6c57c7ef83b0dd1e25c0d9b44aa80](https://github.com/juan-cortes/ledger-live-desktop/pull/14/commits/0266e4e93ff6c57c7ef83b0dd1e25c0d9b44aa80)